File: interpretado-vs-compilados/taller1/python/ejercicio3.py
#Genere 2 arreglos paralelos que representen las sucursales de una empresa y sus ventas. 
# Existen 25 sucursales en la empresa. 
# Presente el promedio de ventas, así como las sucursales con ventas mayores al promedio.

import random

sucursales = []
ventas = []

for i in range(25):
    sucursales.append("Sucursal " + str(i + 1))
    ventas.append(random.randint(1000, 5000))

suma_total = 0
for v in ventas:
    suma_total += v

promedio = suma_total / len(ventas)
print("Promedio de ventas:", promedio)
print("Sucursales que superan el promedio:")

for i in range(len(ventas)):
    if ventas[i] > promedio:
        print(sucursales[i], "-", ventas[i])

File: interpretado-vs-compilados/taller1/c/ejercicio3.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    srand(time(NULL));
    char sucursales[25][20];
    float ventas[25];
    float suma = 0;

    for (int i = 0; i < 25; i++) {
        sprintf(sucursales[i], "Sucursal %d", i + 1);
        ventas[i] = rand() % 9001 + 1000;
        suma += ventas[i];
    }

    float promedio = suma / 25;
    printf("Promedio de ventas: %.2f\n", promedio);
    printf("Sucursales sobre el promedio:\n");

    for (int i = 0; i < 25; i++) {
        if (ventas[i] > promedio) {
            printf("%s: %.2f\n", sucursales[i], ventas[i]);
        }
    }

    return 0;
}